### ISSUE TITLE
Add changelog tab to project pages

### ISSUE THREAD

**Issue by JustLetterV** [modrinth/code#454]
### Is your suggested enhancement related to a problem? Please describe.

No

### Describe the solution you'd like

Changelog tab would be added to project pages similarly to like on the website.

### Additional context

Currently app shows only these tabs on project pages:
![](https://github.com/modrinth/theseus/assets/75657829/ac3082ee-c942-460d-ad88-69af8cba60ed)
On website there is also changelog tab:
![](https://github.com/modrinth/theseus/assets/75657829/ffe0cf4e-0bea-4489-a9ec-729f0da87c24)

Would be nice to have the changelog tab on the app too so reading changelogs wouldn't require switching to the website.

**Comment by Felix14-v2** [comment]
Yes, please! I can't update my mods without making sure that no breaking changes have been made in the new versions, so it is extremely inconvenient to open each version just to read the changelog.
